cmd/envtool/tests_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestShardIntegrationTests(t *testing.T) {
+	t.Parallel()
+
+	var w bytes.Buffer
+	err := shardIntegrationTests(&w, 0, 2)
+	assert.NoError(t, err)
+	assert.NotNil(t, w)
+	assert.Regexp(t, "^\\^.*\\$$", w.String())
+}
+
+func TestShardTests(t *testing.T) {
+	t.Parallel()
+
+	testNames, err := shardTests(0, 2, "integration")
```

Comment:
Thanks for adding test! Great start!
We should also test for `(1, 2, "integration")` index that is not zero as well :hugs: 

Also another good tests are testing for large index `(1000, 10, "integration")` and large total `(10, 1000, "integration")`